docs/serving/using-subroutes.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Creating and using Subroutes"
+linkTitle: "Configuring cluster-local services"
+weight: 20
+type: "docs"
+---
+
+Subroutes are most effective when used with multiple revisions. When defining a knative service/route, the traffic section of the spec can split between the different revisions. For example:
+
+```yaml
+traffic:
+- percent: 0
+  revisionName: foo
+- percent: 40
+  revisionName: bar
+- percent: 60
+  revisionName: baz
+```
+
+This allows anyone targeting the main route to have a 0% chance of hitting revision `foo`, 40% chance of hitting revision `bar` and 60% chance of hitting revision `baz`.
+
+As part of the spec there's a key called `tag`. When a `tag` can be applied to the routes, this will generate an address for the specific traffic target.
```

Comment:
Is "key" the correct term? Would "attribute" be more accurate? (Not sure, but I don't recall seeing the term key used like this before, so I wanted to ask.)

Reply:
"attribute" makes more sense in terms of describing the spec. I'll update it. Thanks!